src/Providers/JWT/Lcobucci.php

```diff
@@ -133,7 +133,7 @@ public function decode($token)
             throw new TokenInvalidException('Could not decode token: '.$e->getMessage(), $e->getCode(), $e);
         }
 
-        if (! $jwt->verify($this->signer, $this->getVerificationKey())) {
+        if ($this->laravel['config']['jwt.verifySignature'] && !$jwt->verify($this->signer, $this->getVerificationKey())) {
```

Comment:
This is not the correct way to do this: configuration options are injected via the constructor, see \Tymon\JWTAuth\Providers\AbstractServiceProvider::registerLcobucciProvider

Reply:
Thanks for your review, I have fixed this. 
This feature is really useful for us, as we forward the token internally among services, while we do not want to do signature verification as it had been done.